pkg/proxy/ipvs/proxier.go

```diff
@@ -574,7 +578,19 @@ func cleanupIptablesLeftovers(ipt utiliptables.Interface) (encounteredError bool
 		}
 	}
 
-	// Flush and remove all of our chains.
+	// Unlink the forwarding chain.
+	args = []string{
+		"-m", "comment", "--comment", "kubernetes forwarding rules",
+		"-j", string(KubeForwardChain),
+	}
+	if err := ipt.DeleteRule(utiliptables.TableFilter, utiliptables.ChainForward, args...); err != nil {
+		if !utiliptables.IsNotFoundError(err) {
+			glog.Errorf("Error removing ipvs Proxier iptables rule: %v", err)
+			encounteredError = true
+		}
+	}
+
+	// Flush and remove all of our chains "-t nat" chains.
```

Comment:
Nit: Flush and remove all our "-t nat" chains.

Reply:
Fixed.